apps/someone-else/views/confirm.html

```diff
@@ -170,8 +170,8 @@
 
       {{<partials-org-type}}{{/partials-org-type}}
 
-      <h2>{{#t}}pages.check-details.footer.title{{/t}}</h2>
-      <p>{{#t}}pages.check-details.footer.explanation{{/t}}</p>
+      <h2>{{#t}}footer.title{{/t}}</h2>
+      <p>{{#t}}footer.explanation{{/t}}</p>
```

Comment:
I'm not sure about the name 'footer'. And I think it should be namespaced with the page name.

maybe `pages.confirm.send-details.title` and  `pages.confirm.send-details.body`